integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/timeseries/TestTimeSeriesCreateTable.scala

```diff
@@ -241,12 +311,12 @@ class TestTimeSeriesCreateTable extends QueryTest with BeforeAndAfterAll {
     assert(e.getMessage.equals("Only one granularity level can be defined"))
   }
 
-  test("test timeseries create table 14: Only one granularity level can be defined 2") {
-    sql("DROP DATAMAP IF EXISTS agg0_second ON TABLE mainTable")
+  test("test timeseries create table 17: Only one granularity level can be defined 2") {
+    sql("DROP DATAMAP IF EXISTS agg0_hour ON TABLE mainTable")
```

Comment:
why change just the name of the table??

Reply:
It's unreasonable to create pre-aggregate with second granularity, second aggregate is used less than hour in reality life.